docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Simple dockerfile example to build a jormungandr and start in genesis mode
+
+FROM ubuntu:cosmic
+LABEL MAINTAINER IOHK
+LABEL description="Jormungandr latest"
+
+ARG PREFIX=/app
+ARG REST_PORT=8448
+
+ENV ENV_PREFIX=${PREFIX}
+
+# prepare the environment
+RUN apt-get update && \
+    apt-get install -y build-essential pkg-config git libssl-dev curl && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# install the node and jcli from source
+RUN mkdir -p ${ENV_PREFIX} && \
+    mkdir -p ${ENV_PREFIX}/src && \
+    mkdir -p ${ENV_PREFIX}/bin && \
+    bash -c "curl https://sh.rustup.rs -sSf | bash -s -- -y" && \
+    export PATH=$HOME/.cargo/bin:$PATH && \
+    rustup install stable && \
+    rustup default stable && \
+    cd ${ENV_PREFIX} && \
+    git clone --recurse-submodules https://github.com/input-output-hk/jormungandr src && \
+    cd src && \
+    git checkout -b last-minutes-updates && \
```

Comment:
Why switch to the new branch?

Reply:
Why not make the branch a --build-arg when you build the docker container:

# default git branch 
ARG BRANCH=master
ENV ENV_BRANCH=${BRANCH}

git checkout -b ${ENV_BRANCH}

During the docker build  you can specify --build-arg BRANCH=last-minutes-updates

Reply:
damn, I forgot to remove it

Reply:
I briefly look at your script btw @onicrom, I just pulled really quickly couple of improvement inspired by your docker, to the one I had locally but clearly this is being thrown together. down the line there's multiple improvements to make to the script, and make multiple variations